signer-npm/src/lib.rs

```diff
@@ -246,13 +257,22 @@ pub fn transaction_sign_lotus(
         .into_serde()
         .map_err(|e| JsValue::from(format!("Error parsing parameters: {}", e)))?;
 
+    let msg = match unsigned_message {
+        MessageTxAPI::Message(m) => m,
+        MessageTxAPI::SignedMessage(_) => {
+            return Err(JsValue::from("Can't serialize SignedMessage"))
```

Comment:
same as earlier, "Attempted to sign a SignedMessage" would make the error clearer